src/sentry/incidents/grouptype.py

```diff
@@ -27,6 +27,8 @@ def build_occurrence_and_event_data(
         self, group_key: DetectorGroupKey, value: int, new_status: PriorityLevel
     ) -> tuple[IssueOccurrence, dict[str, Any]]:
         # Returning a placeholder for now, this may require us passing more info
+        old_status_dict = self.get_state_data([group_key])
```

Comment:
🎉 who knew there were already helper methods for this.

nit on the name:
```suggestion
        previous_state = self.get_state_data([group_key])
```
Since it's not really the "old status" it could be a little confusing, naming it `previous_state` makes line 31 a little bit more clear.